lib/archive/util_test.go

```diff
@@ -727,6 +765,62 @@ func TestAddExclusionsNonNested(t *testing.T) {
 
 }
 
+func TestAddExclusionsTrailingSlash(t *testing.T) {
```

Comment:
Additional ease of review. This test is the same as the below test except for the trailing `/` so we now test for proper behavior for both in the inclusion/exclusion list as well.